ADAL/tests/XCTestCase+TestHelperMethods.h

```diff
@@ -161,6 +163,10 @@
 - (MSIDRefreshToken *)adCreateRefreshToken;
 - (MSIDLegacySingleResourceToken *)adCreateLegacySingleResourceToken;
 
+- (MSIDRequestParameters *)adCreateV2DefaultParams;
```

Comment:
nit: don't we have this functionality in MSID tests?

Reply:
Yes, we do. Let me try to include that MSID test utility class to AD test :)

Reply:
I tried but seems there is no way to include test utility file in common core project to AD test :P

Reply:
Ok, let's leave it that way for now :) I suggest we merge this one.